### DISCUSSION TITLE
Alert payload with webhook integration

### DISCUSSION THREAD

**Discussion by aj231701** [grafana/grafana#45463] [Alerting]
I am working with grafana 8.3.0 and using webhook integration to get alerts. Can anyone help me out to confirm if there are alert payload changes in JSON  since v7.5? or if anyone can share the JSON alerting object/webhook schema specific  for v8.3.0.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
